v2/parser/apis/api/apienc/encoding.go

```diff
@@ -171,6 +171,13 @@ func DescribeResponse(errs *perr.List, responseSchema schema.Type) *ResponseEnco
 		return &ResponseEncoding{}
 	}
 
+	// Check for reserved header prefixes
+	for _, field := range fields[Header] {
+		if strings.HasPrefix(field.WireName, "x-encore") {
```

Comment:
Is the wire name lowercase, or `X-Encore`? Maybe worth lowercasing to be safe? Also maybe should be `x-encore-` and not just `x-encore`